cristin-project/src/test/java/no/unit/nva/cristin/projects/query/projectQuery.feature

```diff
@@ -100,7 +101,7 @@ Feature: API tests for Cristin projects query
     And match contentType == PROBLEM_JSON_MEDIA_TYPE
     And match response.title == 'Bad Request'
     And match response.status == 400
-    And match response.detail == 'Invalid query parameter supplied. Valid parameters: [\'approval_reference_id\', \'approved_by\', \'biobank\', \'funding\', \'funding_source\', \'institution\', \'keyword\', \'language\', \'levels\', \'modified_since\', \'organization\', \'page\', \'participant\', \'project_manager\', \'query\', \'results\', \'sort\', \'status\', \'unit\', \'user\']'
+    And match response.detail == 'Invalid query parameter supplied. Valid parameters: [\'approval_reference_id\', \'approved_by\', \'biobank\', \'depth\', \'funding\', \'funding_source\', \'institution\', \'keyword\', \'language\', \'modified_since\', \'name\', \'organization\', \'page\', \'participant\', \'project_code\', \'project_manager\', \'query\', \'results\', \'sort\', \'status\', \'title\', \'unit\', \'user\']'
```

Comment:
name? title is the one that is supported in cristin. But will both map to title in conversion?

Reply:
name -> title